src/eatery/common_eatery.py

```diff
@@ -101,6 +108,82 @@ def parse_events(event_list, event_date):
     new_events.append(new_event)
   return new_events
 
+def parse_expanded_menu(eatery):
+  """Parses the expanded menu of an eatery.
+
+  Returns a list of FoodCategoryTypes available in the external expandedItems resource
+
+  Args:
+    eatery (dict): A valid json segment from Cornell Dining that contains eatery information
+  """
+  eatery_slug = eatery['slug']
+  menus = requests.get(STATIC_EXPANDED_ITEMS_URL).json()
+  categories = []
+
+  for menu in menus['eateries']:
+    if menu['slug'] == eatery_slug:
```

Comment:
since this is the only use of `eatery_slug`, we can just put `eatery['slug']` in place instead of using a one-time variable